ipv-stub-template.yml

```diff
@@ -21,6 +21,19 @@ Parameters:
     Description: The ARN of the permissions boundary to apply when creating IAM roles
     Default: none
 
+Mappings:
+  EnvironmentConfiguration:
+    dev:
+      ipvStubDomainName: ipvStub.oidc.sandpit.account.gov.uk
```

Comment:
Two things:
- Feels a bit odd to have camel case in a domain name. Is it case sensitive?
- Variable could just be `stubDomain`?

Reply:
Looked into this a bit more - route53 translates `ipvStub` just to `ipvstub` and we have that pattern elsewhere with `orchstub` and `signin` don't know if created a new hosted zone for `stub.oidc` to then create `ipv.stub.oidc..` and `auth.stub.oidc` is worth it.